### PR TITLE
Cards modified

### PR DESCRIPTION
**Pull request by Sapna2001**
Modifications:
- Heights made uniform
- Border added
- Box shadow added

Screenshot:
Annotation 2020-08-16 204723

#6

### CODE REVIEW

**Comment by Sapna2001** [comment]
@abhu-A-J please review this.

**Comment by abhu-A-J** [comment]
> @abhu-A-J please review this.

reduce the width and increase the gap in between cards to accompany that! Rest looks good to me! We'll come up with better designs in future!

File: assets/css/style.css [hunk 1]
```diff
@@ -161,3 +161,27 @@ p {
 		width: 60%;
 	}
 }
+.card{
```
Comment (abhu-A-J) [inline assets/css/style.css:164]: Add some fixed with to fit all content (make cards smaller and changes other params to accompany that)

**Comment by anushbhatia** [comment]
Don't modify the border radius and try adding them maintaining the text unifromity

**Comment by Sapna2001** [comment]
Modifications:-
- Border-radius removed 
- Width reduced
- Space between cards increased.

Annotation 2020-08-16 234230

**Comment by anushbhatia** [comment]
Don't squeeze the size of cards and reduce the z-index it's way too more.Check the responsiveness too.

**Comment by Sapna2001** [comment]
Modifications:-
- box shadow modified
- width changed 

Annotation 2020-08-18 142305

**Comment by smaranjitghose** [comment]
Please fix the merge conflicts

**Comment by anushbhatia** [comment]
don't add z-index

**Comment by Sapna2001** [comment]
Screenshot

Annotation 2020-08-20 100119

**Comment by anushbhatia** [comment]
https___deploy-preview-20--autovaidya netlify app and 5 more pages - Personal - Microsoft​ Edge 20-08-2020 10_39_53
Cards are stretched in mobile view and it's no more responsive too.

**Comment by Sapna2001** [comment]
Mobile View
desktop

Desktop View
mobile

**Comment by Sapna2001** [comment]
Desktop View
desktop

Mobile View
mobile

**Comment by Sapna2001** [comment]
Mobile View
Annotation 2020-08-20 124154

Desktop View
Annotation 2020-08-20 124122

**Comment by anushbhatia** [comment]
https___deploy-preview-20--autovaidya netlify app and 10 more pages - Personal - Microsoft​ Edge 20-08-2020 20_03_01

**Comment by Sapna2001** [comment]
Annotation 2020-08-20 204248

**Pull request merged**